index.php

```diff
@@ -20,4 +20,4 @@
 
 error_reporting(E_ERROR);
 return new BepressImportPlugin();
-?>
\ No newline at end of file
+?>
```

Comment:
As you're modernizing the code, this could be dropped (or perhaps to add the code formatter to the repository) =]